photon-client/src/main/scala/com/linkedin/photon/ml/Constants.scala

```diff
@@ -26,16 +26,16 @@ object Constants {
    *
    * WARNING: This is not visible in println!
    */
-  val DELIMITER = "\u0001"
+  val DELIMITER: String = "\u0001"
```

Comment:
According to our code style settings, IDEA complains that `INTERCEPT_KEY` just a few lines below should have an explicit type. So, we end up with that one `: String`, the others without. Looks non-uniform.